index.js

```diff
@@ -406,12 +415,20 @@ class HtmlWebpackInlineSVGPlugin {
      */
     isNodeValidInlineImage (node) {
 
-        return !!(
-            node.nodeName === 'img'
-            && ((this.inlineAll && _.filter(node.attrs, { name: 'inline-exclude' }).length === 0)
-                || _.filter(node.attrs, { name: 'inline' }).length)
-            && this.getImagesSrc(node))
+        const isInlineAllAndDoesNotHaveInlineExclude = (
+            this.inlineAll &&
+            _.filter(node.attrs, { name: 'inline-exclude' }).length === 0
```

Comment:
Can this be one line please? 😅
```suggestion
            this.inlineAll && _.filter(node.attrs, { name: 'inline-exclude' }).length === 0
```